server/tests-py/validate.py

```diff
@@ -292,6 +292,8 @@ def validate_gql_ws_q(hge_ctx, conf, headers, retry=False, via_subscription=Fals
             assert resp['type'] in ['data', 'error', 'next'], resp
 
     if 'errors' in exp_http_response or 'error' in exp_http_response:
+        if gqlws:
+            resp['payload'] = {'errors':resp['payload']}
```

Comment:
Can we do something better than mutating payload?

Reply:
Could we do something like this?

```suggestion
            resp.update('payload', {'errors': resp['payload']})
```